h2o-algos/src/main/java/hex/ensemble/StackedEnsembleModel.java

```diff
@@ -166,7 +171,57 @@ protected Frame predictScoreImpl(Frame fr, Frame adaptFrm, String destination_ke
     return predictFr;
   }
 
+  /**
+   * Is the indexth model's prediction used in the metalearner?
+   *
+   * @param index
+   */
+  boolean isUsefulBaseModel(int index) {
+    Model metalearner = _output._metalearner;
+    if (metalearner == null) return true; // training phase
+    if (modelCategory == ModelCategory.Multinomial) {
+      // Multinomial models output multiple columns and a base model
+      // might be useful just for one category...
+      int colsPerModel = metalearner._output.nfeatures() / _parms._base_models.length;
+      for (int i = index * colsPerModel; i < (index + 1) * colsPerModel; i++) {
+        if (metalearner.isFeatureUsed(metalearner._output._names[i]))
+          return true;
+      }
```

Comment:
just wondering if columns order is guaranteed during the whole process, and even so, if it's wise to rely on it.
It rather rely just on model keys and column names...
In case of multinomial, is there no way to get the probabilities column names for a given model?

Reply:
Thanks! This helped me realise that I could use column names in multiple other places.